docs/GetStarted/getting_started_windows_bazel.md

```diff
@@ -66,6 +61,7 @@ clone the repository and initialize its submodules:
 > git clone https://github.com/google/iree.git
 > cd iree
 > git submodule update --init
+> python configure_bazel.py
```

Comment:
nit: the text above this "clone the repository and initialize its submodules" could be expanded